Zilon.Core/Zilon.Core/WorldGeneration/Globe.cs

```diff
@@ -67,11 +66,11 @@ public void Save(string path, string realmFileName = null, string branchFileName
                     }
                 }
 
-                realmFileName = realmFileName ?? "realms.bmp";
-                realmBmp.Bitmap.Save(Path.Combine(path, realmFileName), ImageFormat.Bmp);
+                //realmFileName = realmFileName ?? "realms.bmp";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=4940399)